src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Routes.razor

```diff
@@ -0,0 +1,18 @@
+@using BlazorWeb_CSharp.Components.Layout
+
+@*#if (UseWebAssembly) -->
+<Router AppAssembly="@typeof(App).Assembly" AdditionalAssemblies="new[] { typeof(Client.Pages.Counter).Assembly }">
+##else
+<Router AppAssembly="@typeof(App).Assembly">
+##endif*@
+    <Found Context="routeData">
+        <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
+        <FocusOnNavigate RouteData="@routeData" Selector="h1" />
+    </Found>
+    <NotFound>
```

Comment:
How is https://github.com/dotnet/aspnetcore/issues/48983 resolved here? When does this get used?

Reply:
Yeah sorry, that part was still on the todo list. It's implemented now (and the `NotFound` content is gone from here).